scripts/scripts/testing/automated_sheets/Singapore.csv

```diff
@@ -1,86 +1,64 @@
+Singapore,2021-12-24,,207000.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2021-12-25,,207000.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2021-12-26,,207000.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2021-12-27,,207000.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2021-12-28,,208100.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2021-12-29,,208100.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2021-12-30,,208100.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2021-12-31,,208100.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-01,,208100.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-02,,208100.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-03,,208100.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-04,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-05,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-06,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-07,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-08,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-09,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-10,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-11,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-12,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-13,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-14,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-15,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-16,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
+Singapore,2022-01-17,,217500.0,samples tested,https://www.moh.gov.sg/covid-19/statistics,Ministry of Health Singapore,
```

Comment:
Use `cowidev.utils.clean.numbers.metrics_to_num_int` to avoid floats

Reply:
Good point

Reply:
No need to use the method in this case, as it is automatically called when exporting the data with `self.export_datafile`

Reply:
I really should sit down and sit all the util codes!